contracts/near/admin-controlled/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+use near_sdk::env;
+
+pub type Mask = u128;
```

Comment:
Is there any reason to use `u128` for the mask? I think even if we extend it one day to manage any functionality in contracts (not only plausibility) is still too huge. IMO the `u8` is enough for this, though we can extend it to `u16` or `u32` (for future functionality).

Reply:
It is better to be on the safe side. Notice this is generic interface that can be implemented for any contract.